MIP102/MIP102c2-Subproposals/MIP102c2-SP19.md

```diff
@@ -16,14 +17,15 @@ Forum URL: https://forum.makerdao.com/t/mip102c2-sp18-mip-amendment-subproposal/
 
 ## Specification
```

Comment:
Please update the forum URL to https://forum.makerdao.com/t/mip102c2-sp18-mip-amendment-subproposal/22335